scipy/stats/_multivariate.py

```diff
@@ -2642,3 +2644,104 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+class rotation_gen(multi_rv_generic):
```

Comment:
``rotation`` is a much too general name, can you propose something more specific?